pkgs/development/zig-packages/build-zig-package/default.nix

```diff
@@ -0,0 +1,41 @@
+{ lib, stdenv, zig, mktemp }:
+
+{ strictDeps ? true
+, nativeBuildInputs ? [ ]
+, enableParallelBuilding ? true
+, meta ? { }
+
+, zigBuildFlags ? "-Drelease-safe -Dcpu=baseline"
```

Comment:
this will make overriding cumbersome, we should be able to override `-Drelease-safe` and `-Dcpu=baseline` undera different option. That's why I was wanting to wait for zig 0.11, because these flags might change

Reply:
What should we call the flag.